client/src/home/add_button/AddButton.js

```diff
@@ -47,23 +49,44 @@ function AddItemForm(props) {
             <Form.Control
               type="text"
               placeholder="e.g. Apples"
-              onChange={(event) =>
-                setItemName(event.target.value.trim().toLowerCase())
-              }
+              onChange={(event) =>{
+                let val = event.target.value.trim().toLowerCase();
+                if(val.length > 0){
+                  setItemName(event.target.value.trim().toLowerCase());
```

Comment:
The item name should be set regardless of the field's length. The itemName state should always reflect what is in the field, even if the  field is empty.

Reply:
Fixed!